packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -20,8 +20,15 @@ const esSearchStub = sandbox.stub();
 const esScrollStub = sandbox.stub();
 FakeEsClient.prototype.scroll = esScrollStub;
 FakeEsClient.prototype.search = esSearchStub;
+class FakeSearch {
+  static async es() {
+    return new FakeEsClient();
```

Comment:
does this need to be async? Or do we need to await `new FakeEsClient`?

Reply:
👍🏻 : 50b2e5f9a